appveyor.yml

```diff
@@ -18,45 +49,37 @@ install:
+- ps: if (!$env:APPVEYOR_PULL_REQUEST_NUMBER) {$("git checkout -qf " + $env:APPVEYOR_REPO_COMMIT)}
+- ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER) {git fetch -q origin +refs/pull/$($env:APPVEYOR_PULL_REQUEST_NUMBER)/merge; git checkout -qf FETCH_HEAD}
 - ps: git submodule update -q --init --recursive
 
 before_build:
-- ps: $env:playgroundNet46_dir=$Env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46"
-- ps: $env:bundle_dir=$Env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46\ReactAssets"
+- ps: $env:playgroundNet46_dir=$env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46"
+- ps: $env:bundle_dir=$env:APPVEYOR_BUILD_FOLDER + "\ReactWindows\Playground.Net46\ReactAssets"
 - ps: nuget restore ReactWindows\ReactNative.sln
 - npm i -g react-native-cli
 
 build_script:
 - ps: mkdir $env:bundle_dir
 - ps: react-native bundle --platform windows --entry-file $($env:playgroundNet46_dir + "\index.windows.js") --bundle-output $($env:bundle_dir + "\index.windows.bundle") --assets-dest $env:bundle_dir --dev false; echo "Suppressing error"
-- cmd: >-
-    set DEVENV="%VS140COMNTOOLS%\..\IDE\devenv"
-
-    %DEVENV% /build "Debug|x86" ReactWindows\ReactNative.sln
-
-    %DEVENV% /build "ReleaseBundle|x64" ReactWindows\ReactNative.sln
-
-    %DEVENV% /build "Debug|ARM" ReactWindows\ReactNative.sln
+- ps: msbuild /p:Configuration=$env:CONFIGURATION /p:Platform=$env:PLATFORM /p:PlatformToolset=$env:PLATFORM_TOOLSET /nologo /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll" "ReactWindows\ReactNative.sln"
 
 # Start Winium in the background, give it a moment to start
 before_test:
   - ps: $winium = Start-Process -PassThru C:\winium\Winium.Desktop.Driver.exe
   - ps: Start-Sleep -s 5
 
-test:
-  assemblies:
-  - ReactWindows\ReactNative.Net46.Tests\bin\x64\**\*.Tests.dll
-
 test_script:
-#  - npm run flow-check
-  - npm test
+  # Run Nunit 3.x test engine and output to AppVeyor's UI.
+  - ps: nunit3-console "ReactWindows\ReactNative.Net46.Tests\bin\$env:PLATFORM\$env:CONFIGURATION\ReactNative.Net46.Tests.dll" --result=myresults.xml
+  - npm run flow-check
+  # Spec test only works when there is a bundle
+  - ps: "If ($env:CONFIGURATION -Match \"ReleaseBundle\" -Or $env:CONFIGURATION -Match \"DebugBundle\") { npm test }"
 
 after_test:
   - ps: Stop-Process -Id $winium.Id
-  - ReactWindows\packages\OpenCover.4.6.519\tools\OpenCover.Console.exe -register:user -target:"ReactWindows\packages\NUnit.ConsoleRunner.3.5.0\tools\nunit3-console.exe" -targetargs:"ReactWindows\ReactNative.Net46.Tests\bin\x86\Debug\ReactNative.Net46.Tests.dll" -output:ReactWindows_coverage.xml
+  - ReactWindows\packages\OpenCover.4.6.519\tools\OpenCover.Console.exe -register:user -target:"ReactWindows\packages\NUnit.ConsoleRunner.3.5.0\tools\nunit3-console.exe" -targetargs:"ReactWindows\ReactNative.Net46.Tests\bin\$env:PLATFORM\$env:CONFIGURATION\ReactNative.Net46.Tests.dll" -output:ReactWindows_coverage.xml
```

Comment:
pretty sure this won't work for same reason as above. running the ARM assembly will fail since the container is x64.